examples/README.md

```diff
@@ -866,6 +866,18 @@ spec:
         httpGet:
           path: /ping
           port: 8086
+
+  - name: influxdb-client
+    inputs:
+      parameters:
+      - name: cmd
+    container:
+      image: appropriate/curl:latest
+      command: ["/bin/sh", "-c"]
+      args: ["{{inputs.parameters.cmd}}"]
+      resources:
+        memory: 32Mi
+        cpu: 100m
```

Comment:
Thanks for contributing! 

This made me spot a bug in our examples. The resources field should have either `requests` or `limits` under it. e.g.
```
      resources:
        requests:
          memory: 32Mi
          cpu: 100m
```
Do you mind applying that change to this README as well as the examples/influxdb-ci.yaml?

Reply:
Thanks for the quick reply Jesse. I have added the changes you requested.